common/lib/dependabot/file_updaters/vendorable.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Dependabot
+  module FileUpdaters
+    module Vendorable
+      private
+
+      # Returns changed files in the vendor/cache folder
+      #
+      # @param base_directory [String] Update config base directory
+      # @return [Array<Dependabot::DependencyFile>]
+      def updated_vendor_cache_files(base_directory:)
+        return [] unless repo_contents_path && vendor_dir
```

Comment:
This is awesome! Nit style ☕ 🚗 review, thoughts on using this as a service-class/object or passing in `repo_contents_path` `vendor_dir` as args here? Get a bit nervous when mixins reference things from the parent class, so easy to miss

Reply:
Yeah, good idea 👍  maybe something like `Dependabot::FileUpdaters::VendorUpdater.new(repo_contents_path:, vendor_dir:)`?